src/PKSim.Core/Services/IndividualEnzymeFactory.cs

```diff
@@ -37,11 +40,17 @@ public IndividualEnzyme AddUndefinedLiverTo(Individual individual)
          CoreConstants.Compartment.LiverZones.Each(zoneName =>
          {
             var zone = liver.Container(zoneName);
-            AddContainerExpression(zone.Container(CoreConstants.Compartment.INTRACELLULAR), undefinedLiver.Name,
-               RelExpParam(REL_EXP, defaultValue: 1),
+            var intracellular = zone.Container(CoreConstants.Compartment.INTRACELLULAR);
+            AddContainerExpression(intracellular, undefinedLiver.Name,
+               RelExpParam(REL_EXP),
                FractionParam(FRACTION_EXPRESSED_INTRACELLULAR, CoreConstants.Rate.ONE_RATE),
                InitialConcentrationParam(CoreConstants.Rate.INITIAL_CONCENTRATION_INTRACELLULAR)
             );
+
+            var relExpParameter = intracellular.EntityAt<IParameter>(undefinedLiver.Name, REL_EXP);
```

Comment:
@Yuri05 Following up on your comments, I am updating the value of the parameter after the fact .No more messing around